projects/go-lib/src/lib/components/go-header-bar/go-header-bar.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "../../../styles/_variables.scss";
+.go-header-bar {
+  margin: -2rem;
+  height: 68px;
+  background: white;
+  box-shadow: 0 0 2px rgba(0, 0, 0, 0.12), 0 2px 2px rgba(0, 0, 0, 0.24);
```

Comment:
We should try to utilize one of the `box-shadow` variables for this. Check the available variables in the main `variables.scss` file to see if any of them would work. If none of them match, check with UX to see which shadow we should use.